OpenEdXMobile/res/values/strings.xml

```diff
@@ -584,6 +584,8 @@
     <string name="profile_incomplete_about_me">No “About me” info provided.</string>
     <!-- Window title for edit profile screen -->
     <string name="edit_user_profile_title">Edit profile</string>
+    <!-- Content description for edit profile username -->
+    <string name="edit_user_username_content_description">Your username is {username}</string>
```

Comment:
Shouldn't this just use the `profile_name_description` string, which is used in the read-only profile page, and is just "Name: {name}"?